.github/workflows/software_repo_native.yml

```diff
@@ -25,11 +25,9 @@ jobs:
             persist-credentials: false
 
         - name: Mount EESSI CernVM-FS software repository
-          uses: cvmfs-contrib/github-action-cvmfs@55899ca74cf78ab874bdf47f5a804e47c198743c # v4.0
+          uses: eessi/github-action-eessi@v3
```

Comment:
I'd suggest to use the latest version (v3.1) of our github action. See https://github.com/EESSI/github-action-eessi/compare/v3.0.0...v3.1.0

Reply:
Leaving it lose implies using the latest `v3` release (so `v3.1.0` today and `v3.2.0` when it comes out)

Reply:
This is actually the reason that referencing GitHub Actions by tag is insecure, since you change the commit the tag points to at any point.

Reply:
Didn't know that. Thanks for clarifying it. Would we want to use then a specific commit instead of `v3`?

Reply:
It's our own action and one of the few places we use it, it's a good thing to actually see whether it breaks on new releases